tictactoe/src/main/java/com/github/martingaston/tictactoe/api/API.java

```diff
@@ -11,9 +11,9 @@ public static String init(String mode) throws IOException {
         return Encode.from(Response.initial(mode));
     }
 
-    public static String update(int position, String jsonString) throws IOException {
-        GameJSON previousGame = Decode.from(jsonString);
-        GameJSON updated = Update.from(position, previousGame);
+    public static String update(String jsonString) throws IOException {
```

Comment:
`jsonString` is quire redundant as a name as your type specify that's the json is a string

Reply:
That is very true!